src/style.css

```diff
@@ -1,3 +1,85 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.logo-container {
+  width: 40px;
+  height: 40px;
+  position: absolute;
+  top: 10px;
+  left: 50px;
+}
+
+.logo-container img {
+  width: 100%;
+  height: auto;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  color: rgba(0, 0, 0, 1);
+  display: flex;
+  align-items: center;
+  padding: 0;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+.nav__link:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.nav__link.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: -5px;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  border-radius: 8px;
+  background-color: rgba(0, 172, 220, 1);
+}
+
+.nav__container {
```

Comment:
Be cautious when using fixed `padding-left` values as they can cause layout issues on smaller screens. Consider using responsive units or CSS Grid/Flexbox properties to manage spacing.